GUI/Types/Renderer/ShapeSceneNode.cs

```diff
@@ -266,8 +266,6 @@ public override void Render(Scene.RenderContext context)
             GL.UseProgram(renderShader.Program);
 
             renderShader.SetUniform4x4("transform", Transform);
-            renderShader.SetUniform1("bAnimated", false);
-            renderShader.SetUniform1("sceneObjectId", Id);
```

Comment:
keep the ids

Reply:
I found these lines to be useless because the shader does not use bone animation at all. I guess it helps with `context.ReplacementShader`, weird design though.